src/main/java/duke/Exceptions/DukeException.java

```diff
@@ -0,0 +1,27 @@
+package duke.exceptions;
+
+/**
+ * Exceptions of Duke object, errors which duke is likely to encounter and what is done
+ * to catch and deal with the errors
+ */
+public class DukeException extends Throwable {
```

Comment:
I love how you have many different specific Duke execeptions that extend "DukeExeception".